avocado/utils/pmem.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, ndctl='ndctl', daxctl='daxctl'):
         if not abs_ndctl:
             raise PMemException("Cannot use library without "
                                 "proper ndctl binary")
-        self.ncdctl = abs_ndctl
+        self.ndctl = abs_ndctl
```

Comment:
Although you can have `self.ndctl` assigned on line 55 and discard this line, I don't have a problem with this patch.

Quick question, on line 61 the same pattern of assign, check, assign to self happens, but on line 65 there is an assignment of the binary and not of the absolute path. Is that correct?

Reply:
> Although you can have `self.ndctl` assigned on line 55 and discard this line, I don't have a problem with this patch.
> 
> Quick question, on line 61 the same pattern of assign, check, assign to self happens, but on line 65 there is an assignment of the binary and not of the absolute path. Is that correct?

No, that is also a mistake. Sign! My bad, and this LGTM, sending a fix for the other assignment ASAP.